pkgs/top-level/python-packages.nix

```
@@ -7310,6 +7310,8 @@ in {
 
   inflect = callPackage ../development/python-modules/inflect { };
 
+  iterfzf = callPackage ../development/python-modules/iterfzf { };
```

incase a `fzf` package gets added to python scope
```suggestion
  iterfzf = callPackage ../development/python-modules/iterfzf {
    inherit (pkgs) fzf;
  };
```